pkg/controller/servicebindingrequest/servicebinder.go

```diff
@@ -371,6 +377,24 @@ func (b *serviceBinder) setApplicationObjects(
 	sbrStatus.Applications = boundApps
 }
 
+// set default value for application selector
+func ensureDefaults(applicationSelector *v1alpha1.ApplicationSelector) {
+	if applicationSelector.LabelSelector == nil {
+		applicationSelector.LabelSelector = &metav1.LabelSelector{}
+	}
+	if applicationSelector.BindingPath == nil {
+		applicationSelector.BindingPath = &v1alpha1.BindingPath{
+			ContainersPath: defaultPathToContainers,
+		}
+	} else {
+		if applicationSelector.BindingPath.ContainersPath == "" && applicationSelector.BindingPath.SecretPath != "" {
+			applicationSelector.BindingPath = &v1alpha1.BindingPath{
```

Comment:
Isn't this assignment overwriting `bindingPath.secretPath` which has been defined by the user (and part of the conditional statement in L390?

Reply:
Fixed here. 9e875534459af740b5ac4a7b3c8e4cc48bd60069